examples/preprocessing/plot_muscle_detection.py

```diff
@@ -0,0 +1,85 @@
+"""
+===========================
+Annotate muscle artifacts
+===========================
+
+Muscle contractions produce high frequency activity that can contaminate the
+brain signal of interest. Muscle artifacts can be produced when clinching the
+jaw, swalloing, or twitching a head muscle. Muscle artifacts are most notable
+in the range of 110-140Hz.
+
+This example uses `annotate_muscle` to annotate segments where muscle activity
+likely occurred. This is done by band-pass filtering the data in the 110-140 Hz
+range. Then, the envelope is taken to account for peaks and troughs. The
+envelope is z-scored and averaged across channels. To remove noisy transient
+peaks, the z-scored average is low-pass filtered to 4 Hz. Segments above a set
+threshold are annotated as BAD_motion. In addition, `min_length_good` allows to
+discard god segments of data between bad segments that are to transient.
```

Comment:
```suggestion
threshold are annotated as ``BAD_muscle``. In addition, the ``min_length_good``
parameter determines the cutoff for whether short spans of "good data" in between
 muscle artifacts are included in the surrounding "BAD" annotation.
```

Note that I'm suggesting the annotation change from ``BAD_motion`` to ``BAD_muscle``, since it's more similar to the function name, and (I think?) "motion" should be reserved for talking about head motion detection / correction.  This will necessitate code changes, not just changing the text here.

Reply:
Sorry, that was a typo, should have been BAD_muscle